CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+All notable changes to this project will be documented in this file.
+
+<a id="2_25_0"></a>
+## [2.25.0] - 2025-11-12
+
+### Added
+
+- Implement drag-and-drop to reorder accounts ([#5590](https://github.com/deltachat/deltachat-desktop/issues/5590))
+- New broadcast channel behaviour (experimental) ([#5686](https://github.com/deltachat/deltachat-desktop/issues/5686))
+- A11y: Reply and Attachment landmark
+
+### Changed
+
+- Upgrade core to 2.25 ([#5665](https://github.com/deltachat/deltachat-desktop/issues/5665))
+- Change link color ([#5701](https://github.com/deltachat/deltachat-desktop/issues/5701))
+- Remove "Watch Sent Folder" preference
+- Update classic email login wordings ([#5688](https://github.com/deltachat/deltachat-desktop/issues/5688))
+- Do not use HTTPS request for default instant onboarding ([#5618](https://github.com/deltachat/deltachat-desktop/issues/5618))
+- Remove address from Vcard ([#5672](https://github.com/deltachat/deltachat-desktop/issues/5672))
+- Hide or move email address ([#5700](https://github.com/deltachat/deltachat-desktop/issues/5700))
```

Comment:
I think such short messages don't make sense to users. Maybe we should be manually adjusting the changelog entries after they have been generated? Or is this too much hassle?

The same goes for more technical entries like "Lowercase scheme before comparison", this means nothing to end users.